src/types/keyboard.zig

```diff
@@ -97,4 +97,21 @@ pub const Keyboard = extern struct {
     pub fn getModifiers(keyboard: *Keyboard) ModifierMask {
         return @bitCast(ModifierMask, wlr_keyboard_get_modifiers(keyboard));
     }
+
+    extern fn wlr_keyboard_notify_modifiers(
+        keyboard: *Keyboard,
+        mods_depressed: u32,
+        mods_latched: u32,
+        mods_locked: u32,
+        group: u32,
+    ) void;
+    pub fn notifyModifiers(keyboard: *Keyboard, modifiers: Modifiers) void {
```

Comment:
It doesn't seems there is a need for a wrapper here

`pub const notifyModifiers = wlr_keyboard_notify_modifiers;`

Reply:
this wrapper makes calling the function a lot nicer if the caller already has a modifiers struct. If the caller doesn't, then creating the struct inline also helps avoid mixing up the four separate u32 parameters. I like the wrapper here.